.dockerignore

```diff
@@ -1,5 +1,4 @@
 example
-tests
```

Comment:
The `harmonyservices/harmony-gdal-adapter-test` Docker image needs to be able to find and copy the `tests` directory.